ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationIF.ts

```diff
@@ -0,0 +1,72 @@
+import { HomeSectionIF } from '@/interfaces'
+
+export interface MhrRegistrationIF {
+  clientReferenceId: string,
+    declaredValue: string,
+    registeringParty: {
```

Comment:
In the API registeringParty will be renamed to submittingParty and will have phoneNumber and phoneExtension properties.

Reply:
Awesome, i recall this being mentioned now. 
Will update, thank you!